sernet.gs.service/src/sernet/verinice/service/commands/RemoveElement.java

```diff
@@ -319,43 +309,33 @@ private void removeConfiguration(CnATreeElement person) throws CommandException
             getCommandService().discardUserData();
         }
     }
-    
-    private boolean isOrphanEntity(Entity entity) {
-        DetachedCriteria crit = DetachedCriteria.forClass(CnATreeElement.class);
-        crit.setFetchMode("entity", FetchMode.JOIN);
-        crit.add(Restrictions.eq("entity.dbId", entity.getDbId()));
-        
-        IBaseDao dao = getDaoFactory().getDAOforTypedElement(element);
-        List<CnATreeElement> entities = dao.findByCriteria(crit);
-        return entities.size() > 1 ? false : true;
-    }
-
-    private void deleteOrphanEntity(Entity entity) {
-        IBaseDao<Entity, Serializable> entityDao = getDaoFactory().getDAO(Entity.class);
-        if (getLog().isInfoEnabled()) {
-            getLog().info("Found and deleted orpahn entity with dbId: " + entity.getDbId());
-        }
-        entityDao.delete(entity);
-    }
 
-    /* (non-Javadoc)
+    /*
+     * (non-Javadoc)
+     * 
```

Comment:
Please don't use `(non-Javadoc)`. The block comment also makes this clear by opening with one asterisk.

Try
```sh
git diff your/branch custom1 | grep non-Javadoc
```
or similar to find similar code touched by you.

More confusing, you are using `@see` here.